Tools/TestWebKitAPI/Info.plist

```diff
@@ -30,5 +30,10 @@
 	<string>${PRODUCT_NAME}</string>
 	<key>CFBundleIdentifier</key>
 	<string>${PRODUCT_BUNDLE_IDENTIFIER}</string>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
+		<true/>
+	</dict>
```

Comment:
Is this needed for the API test?

Reply:
Yes, I added a comment in the commit message, too.